tests/Integration/Integration.Tests/PurchaseOrderModuletests/WhenGettingPurchaseOrder.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Linn.Common.Facade;
```

Comment:
using should be inside the namespace. Also the casing on the namespace is correct but the folder has a lowercase t in tests